ログアウトページとクレジットカード一覧画面の実装

zerotaisuke opened the pull request:
# what

　ログアウトページとクレジット一覧画面を作成した。
・三輪さんに作成いただいたマイページ画面(hamlとcssを)を活用。
・ログアウト、クレジット一覧画面でもメディアクエリが機能するようにした。

Commit:
Merge branch 'ryo-kamido/master'

Commit:
Merge branch 'ryo-kamido/master'

Commit:
Merge branch 'ryo-kamido/master'

Commit:
Merge remote-tracking branch 'ryo-kamido/master'

Commit:
Merge remote-tracking branch 'ryo-kamido/master'

Commit:
Merge remote-tracking branch 'ryo-kamido/master'

Commit:
Basic認証の復旧

スペースを追記

Commit:
Merge remote-tracking branch 'ryo-kamido/master' into miwaさんのページと連携

Commit:
クレジット一覧画面

# what
  クレジット一覧画面の作成

# why
　クレジット一覧画面の実装

Commit:
ルーティングの編集

# what
クレジットの確認画面へのルーティングを編集
new→credit2
logout→logout

# why
認識しやすくするため

Commit:
ログアウトページの作成

# what
ログアウトページの作成

zerotaisuke commented:
<img width="1440" alt="スクリーンショット 2019-03-12 16 54 43" src="https://user-images.githubusercontent.com/47076525/54183490-a18cbc80-44e7-11e9-87f5-e450c84c2954.png">
<img width="1440" alt="スクリーンショット 2019-03-12 16 54 30" src="https://user-images.githubusercontent.com/47076525/54183491-a2255300-44e7-11e9-8c7c-71b504d60831.png">

zerotaisuke commented:
ご確認のほどよろしくお願いいたします。

hiroki5123 commented on app/views/profiles/credit2.html.haml:
```diff
@@ -0,0 +1,26 @@
+                  %li.l-chapter__creditlist
+                    %form.l-chapter__creditcard
+                      %figure
+                        %img{:alt => "visa", :height => "15", :src => "//www-mercari-jp.akamaized.net/assets/img/card/visa.svg?2845599746", :width => "49"}
```
image_tagで書きましょう
スタイルはCSS側に指定しましょう

zerotaisuke replied:
image_tagで記載し、スタイルをcssに指定しました。

hiroki5123 commented on app/views/profiles/credit2.html.haml:
```diff
@@ -0,0 +1,26 @@
+                      .l-chapter__creditdate **/**
+                      %button.l-chapter__button{:type => "submit"} 削除する
+          .l-chapter__footer
+            %a.l-chapter__cash{href:"#"}
```
link_toで仮置きしましょう

zerotaisuke replied:
仮置きしました

Commit:
指摘いただいた点を修正

# what
・コメントアウトの削除
・image_tagへ修正
・link_toへ修正
・スタイル記述をcssに
・assets/images/フォルダ内にcreditlogファイルの作成

zerotaisuke commented on app/assets/stylesheets/profile/_credit.scss:
```diff
@@ -0,0 +1,99 @@
+.original-clearfix{
+  // width: 700px;
```
削除しました

zerotaisuke commented on app/assets/stylesheets/profile/_credit.scss:
```diff
@@ -0,0 +1,99 @@
+
+.content{
+  width: 700px;
+  // float: none;
```
削除しました

zerotaisuke commented:
ありがとうございました
